config/testing.toml

```diff
@@ -62,3 +62,4 @@ level = 4
 
 [board]
 cycle_at = 15
+api_url = "https://api.thehellisthis.com"
```

Comment:
I'm okay with this being added here.

Reply:
TODO: replace with localhost / CI hosted target